contrib/docker-device-tool/README.md

```diff
@@ -0,0 +1,11 @@
+Docker device tool for devicemapper storage driver backend
+===================
+
+The ./contrib/docker-device-tool contains a tool to manipulate devicemapper thin-pool.
+
+Compile
+========
+
+    $ make shell
+    ## inside build container
+    $ go build -tags libdm_no_deferred_remove contrib/docker-device-tool/device_tool.go
```

Comment:
Because the devicemapper version installed in image does not contain this flag.

```
root@1903dadfb993:/go/src/github.com/docker/docker# go build  contrib/docker-device-tool/device_tool.go
# github.com/docker/docker/pkg/devicemapper
could not determine kind of name for C.dm_task_deferred_remove
root@1903dadfb993:/go/src/github.com/docker/docker# go build -tags libdm_no_deferred_remove contrib/docker-device-tool/device_tool.go
root@1903dadfb993:/go/src/github.com/docker/docker#
```